sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/windowing/Trigger.java

```diff
@@ -263,13 +264,16 @@
     public abstract MergingTriggerInfo trigger();
   }
 
-  @Nullable
-  protected final List<Trigger> subTriggers;
+  public final List<Trigger> subTriggers;
```

Comment:
All the triggers that actually have subtriggers need the list to be public. In a world of object algebras, this would be pushed down into the subclasses, not in the superclass. I didn't want to make that change just yet. I could make certain subclasses that need it override it with looser access, though.